easybuild/easyconfigs/s/spam/spam-0.6.1.1-foss-2021a.eb

```diff
@@ -0,0 +1,77 @@
+easyblock = 'PythonBundle'
+
+name = 'spam'
+version = '0.6.1.1'
+
+homepage = 'https://ttk.gricad-pages.univ-grenoble-alpes.fr/spam/'
+description = """This is a Python package for handling and correlating 3D fields for applications in material science.
+Spam has evolved to cover needs of data analysis from 3D x-ray tomography work and correlated random fields with
+mechanical applications."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('Eigen', '3.3.9'),
+]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('PyQt5', '5.15.4'),
+    ('h5py', '3.2.1'),
+    ('scikit-image', '0.18.3'),
+    ('PyAMG', '4.2.3'),
+    ('matplotlib', '3.4.2'),
+    ('CGAL', '4.14.3'),
+    ('gmsh', '4.9.0'),
+    ('libxml2', '2.9.10'),
+    ('libxslt', '1.1.34'),
+    ('ICU', '69.1'),
+    ('GMP', '6.2.1'),
+    ('MPFR', '4.1.0'),
+    ('netCDF', '4.8.0'),
+    ('pybind11', '2.6.2'),
+    ('SimpleITK', '2.1.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('qimage2ndarray', '1.9.0', {
+        'checksums': ['a79075c6d0d8443c4422ee969a5409e44fd0254571604ccea875c6c95e3f55ea'],
+    }),
+    ('python-decouple', '3.6', {
+        'modulename': 'decouple',
+        'checksums': ['2838cdf77a5cf127d7e8b339ce14c25bceb3af3e674e039d4901ba16359968c7'],
+    }),
+    ('tifffile', '2022.5.4', {
+        'checksums': ['b03147a15862b7c1d90d47435197f149bef7a52c25ad67cf1f9b465faa71b8d2'],
+    }),
+    ('progressbar', '2.5', {
+        'checksums': ['5d81cb529da2e223b53962afd6c8ca0f05c6670e40309a7219eacc36af9b6c63'],
+    }),
+    ('commonmark', '0.9.1', {
+        'checksums': ['452f9dc859be7f06631ddcb328b6919c67984aca654e5fefb3914d54691aed60'],
+    }),
+    ('rich', '12.4.4', {
+        'checksums': ['4c586de507202505346f3e32d1363eb9ed6932f0c2f63184dea88983ff4971e2'],
+    }),
+    ('meshio', '5.3.4', {
+        'checksums': ['e240692d7ff279c12b13c6e5d7b403629a86ad2b44e9230958b3f00c1ec369f0'],
+    }),
+    (name, version, {
+        'patches': ['spam-0.6.1.1_loosen_requirements.patch'],
```

Comment:
@Micket Wouldn't it be easier w.r.t. maintenance to use a `sed` command to loosen up the strict version requirements?

```python
'preinstallopts': "sed -i 's/==/>=/g' requirements.txt && ",
```

That still leaves the commenting out of `gmsh` (which is motivated) and `rpy2` (unclear why?)

Reply:
@Micket Ping